JumpscaleCore/servers/threebot/ThreeBotServersFactory.py

```diff
@@ -58,26 +58,8 @@ def local_start_default(self, web=True, packages_add=False, timeout=120):
             self.install()
             self.default.stop()
 
-        # will return client
         client = self.default.start(background=True, web=web, timeout=timeout)
 
-        client.actors.package_manager.package_add(path=j.threebot.package.phonebook._dirpath)
-
-        if packages_add:
```

Comment:
Did you check that nothing depends on this? And if `packages_add` is not used any anymore, why is it still in the custom signature?